ionic_user_interface/src/components/gym-selector/GymRequestForm.vue

```diff
@@ -97,7 +97,7 @@ export default defineComponent({
     const alertSubmissionSuccess = async () => {
       const alert = await alertController.create({
         header: 'Request Submitted',
-        subHeader: 'Thank you! We will respond to you in 12 hours',
+        subHeader: 'Thank you! A response email will be sent in 3 to 6 hours',
```

Comment:
Same comment, seems a little too short of a time frame. I suggest 2-12 hours. Also, are we gonna manually email them back?